monkey/agent_plugins/payloads/cryptominer/manifest.yaml

```diff
@@ -0,0 +1,19 @@
+name: Cryptominer
```

Comment:
I'm wondering if we should rename all of this "Cryptojacker".

Reply:
`When cryptomining takes place without the permission of the infrastructure's owners, it becomes cryptojacking.`

Probably yep.